packages/@lwc/state/package.json

```diff
@@ -11,11 +11,15 @@
   "main": "dist/index.js",
   "scripts": {
     "build": "tsc",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "vitest"
```

Comment:
I think I'm fine with `vitest` for now. I'm not sure if we'll end up with integration tests with LWCs and, if so, it'll be nice to have a browser testing environment. If we don't go that direction, though, it might be worth simplifying things by just using the build-in Node test runner.